src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -668,6 +688,73 @@ InferenceEngine::Precision FullyConnected::getRuntimePrecision() const {
     return getMaxPrecision(inputPrecisions);
 }
 
+bool FullyConnected::canUseSparseWeights() {
+    if (useSparseWeights == false) {
```

Comment:
I believe we don't need this condition anymore given EXPERIMENTAL_FC_SPARSE_WEI_DECOMPRESSION_ENABLE 
 will be deleted

Reply:
removed, thanks